packages/htm/src/render.ts

```diff
@@ -1,65 +1,38 @@
 import baseHtm from 'htm';
+import {isRemoteComponent} from '@remote-ui/core';
 import type {RemoteRoot, RemoteComponent} from '@remote-ui/core';
 
-interface Element {
-  type: string;
-  props: Record<string, any>;
-  children: (string | Element)[];
-}
+export function createRender(root: RemoteRoot<any, any>) {
+  return baseHtm.bind((type, props, ...children) => {
+    const normalizedChildren: (string | RemoteComponent<any, any>)[] = [];
```

Comment:
Just stylistic, but especially when currying like this, consider using `=>` for both functions and drop the extra `{ }`?  Prettier is good with line breaks to make it readable.

```ts
const createRender = (root: RemoteRoot<any, any>) =>
  baseHtm.bind((type, props, ...children) => {
    const normalizedChildren: (string | RemoteComponent<any, any>)[] = []; 
    // ...
  });

```

Reply:
I think this is stylistic, my preference is strongly for function declarations, especially at module scope, since they get hoisted. I don't feel too strongly about this but I don't see a ton of value in trying to make the code more terse (personally, I find nested array function returns to be quite hard to read, though I know this is not universally true for all developers)